<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BasharPC" failures="1" tests="3" name="cuccumbersOptions.TestNGRunner" time="31.795" errors="0" timestamp="2024-06-18T08:58:28 PDT" skipped="0">
  <testcase classname="cuccumbersOptions.TestNGRunner" name="runScenario" time="12.264"/>
  <testcase classname="cuccumbersOptions.TestNGRunner" name="runScenario" time="12.257">
    <failure message="expected [ready] but found [finished]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [ready] but found [finished]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.FileEncryptionAndDecryptionTest.The_user_clicks_the_Decrypt_button_to_encrypt_the_file_1(FileEncryptionAndDecryptionTest.java:195)
at âœ½.The user clicks the Decrypt button to decrypt the file correctly(file:///C:/Users/basha/Downloads/crypto-testing/src/test/java/features/FileEncryptionAndDecryptionTest.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="cuccumbersOptions.TestNGRunner" name="runScenario" time="7.274"/>
</testsuite> <!-- cuccumbersOptions.TestNGRunner -->
